python/tests/unit/arcticdb/version_store/test_aggregation_dynamic.py

```diff
@@ -93,8 +96,8 @@ def test_hypothesis_mean_agg_dynamic(lmdb_version_store_dynamic_schema_v1, df):
         index=range_indexes(),
     )
 )
-def test_hypothesis_sum_agg_dynamic(s3_version_store_dynamic_schema_v2, df):
-    lib = s3_version_store_dynamic_schema_v2
+def test_hypothesis_sum_agg_dynamic(lmdb_version_store_dynamic_schema_v1, df):
```

Comment:
Why was the encoding version changed from v2 to v1?

Reply:
typo, fixed